jsk_spot_robot/spoteus_demo/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(spoteus_demo)
```

Comment:
can we put this file under `spoteus/demo` ?

Reply:
I will put them under spoteus.